docs/api/tenants.md

```diff
@@ -0,0 +1,109 @@
+# Tenants
+
+Tenants can be managed using the REST API. Only users with administrative
+powers can perform these calls.
+
+## List all tenants
+
+### URL
+
+```http
+GET /tenants HTTP/1.1
+```
+
+### Response
+
+The response contains a list of organisations.
```

Comment:
It contains a list of organisations, or a list of tenants? I would say "tenants" here.